fastplotlib/graphics/selectors/_linear.py

```diff
@@ -111,6 +112,9 @@ def __init__(
         edge_color: str | tuple | np.ndarray, default "w"
             color of the selector
 
+        hitbox_width: float, default 14.0
+            the width around the selector which is responsive to mouse events
```

Comment:
```suggestion
            the width around the selector which is responsive to mouse events, in logical pixels.
```

Reply:
I think I would call it *extra* width`.

Another option is to think of `hitbox_width` as a margin and doing `thickness=thickness + 2 * hitbox_width`. But I think the current implementation is easier to explain as an extra width on top of the normal width.